workflows/video-generation-workflow/README.md

```diff
@@ -0,0 +1,45 @@
+# Spell Workflow Example: Video Generation
+This is an example of how to use the Spell Python API and a workflow to train a [pix2pix](https://github.com/affinelayer/pix2pix-tensorflow) model and generate videos with it.
+
+For more information regarding Spell, the Python API, runs, and workflows,
+check out the [Spell documentation](https://spell.run/docs).
+
+To run this code you first need to [sign up for a Spell account](https://web.spell.run/register)
+and install Spell: `pip install spell`.
+
+# Introduction
+Pix2pix is an image-to-image transformation method. Given a video, we can train a pix2pix model to predict the next frame based on current frame. After training, such a model could be used to generate a video starting from only one frame. In the generation process, every newly created frame would be fed back to the model to predict the next one.
+
+# Run it
+
+### 1. Clone this repo:
+```ShellSession
+$ git clone https://github.com/spellrun/spell-examples.git
+```
```

Comment:
You really only need a single `workflow.py` python file (and the .mp4 video too). Might make sense to explain that in this step with a quick `note`